apps/ml-service/venv/lib/python3.10/site-packages/dateutil/tzwin.py

```
@@ -0,0 +1,2 @@
+# tzwin has moved to dateutil.tz.win
+from .tz.win import *
```

Ruff (0.11.9)

`from .tz.win import *` used; unable to detect undefined names

(F403)